apps/wpcom-block-editor/src/calypso/iframe-bridge-server.js

```diff
@@ -560,6 +560,26 @@ function openLinksInParentFrame() {
 	}
 }
 
+/**
+ * Ensures the Calypso Customizer is opened when clicking on the the FSE blocks' edit buttons.
+ *
+ * @param {MessagePort} calypsoPort Port used for communication with parent frame.
+ */
+function openCustomizer( calypsoPort ) {
+	const customizerLinkSelector = 'a.components-button[href*="customize.php"]';
+	$( '#editor' ).on( 'click', customizerLinkSelector, e => {
```

Comment:
Why do we use this selector filtering, instead of using `customizerLinkSelector` as the primary selector?

Reply:
The above binds the click event handler to the editor div. So if the other selector is dynamically added we don't need to add a click event on the child when it's created.

See also http://api.jquery.com/delegate/

Reply:
Ah! I knew I was missing something – I get it now 👍